Tests/designspaceLib/designspace_v5_test.py

```diff
@@ -185,6 +185,22 @@ def test_read_v5_document_simple(datadir):
                 styleName="Supports",
                 localisedFamilyName={},
             ),
+            SourceDescriptor(
+                filename="masters/masterTest2.ufo",
+                path=posix(str((datadir / "masters/masterTest2.ufo").resolve())),
+                name="master.ufo3",
+                layerName=None,
+                location={"Italic": 1.0, "Weight": 0.0, "Width": 100.0},
+                copyLib=False,
```

Comment:
why do we keep initializing these deprecated attributes?

Reply:
Because the `True` ones are needed for equality testing.